### PR TITLE
Gitignore

### PR DESCRIPTION
**Pull request by Zerocreative0815**
added the .gitignore file that prevents adding files/folders that are specific to IntelliJ IDEA and Maven

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.idea
+
+out/*
```
Comment (nipafx) [inline .gitignore:3]: The `target` folder that MAven will create is missing.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@
 #Bene, Leon, Peter and Janos
 
 code, comments and conversation will be in english
+
```
Comment (nipafx) [inline README.md:7]: This change does not belong into this pull request.

**Review by nipafx — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Zerocreative0815) [reply .gitignore:3]: Yes... because it's not online yet...i will update it later then.

Comment (nipafx) [reply .gitignore:3]: No, that's fine. You can leave it like this and then the person creating the Maven project adds the line.

Comment (Zerocreative0815) [reply .gitignore:3]: Yes. someone will do it :)

**Pull request closed without merging**